addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="avmfritz"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Supported Fritz! devices and features -->
+    <thing-type id="FRITZ_DECT_200">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzbox" />
+		</supported-bridge-type-refs>
+
+        <label>FRITZ!DECT 200</label>
+        <description>FRITZ!DECT200 switchable outlet</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature" />
+			<channel id="energy" typeId="energy" />
+			<channel id="power" typeId="power" />
+			<channel id="outlet" typeId="outlet" />
+		</channels>
+
+		<config-description>
+			<parameter name="ain" type="text" required="true">
+				<label>AIN</label>
+				<description>The AHA id (AIN) that identifies one specific device.</description>
+			</parameter>
+		</config-description>
+    </thing-type>
+
+    <thing-type id="FRITZ_Powerline_546E">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzbox" />
+		</supported-bridge-type-refs>
+
+        <label>FRITZ!Powerline 546E</label>
+        <description>FRITZ!Powerline 546E with switchable outlet</description>
+
+		<channels>
+			<channel id="energy" typeId="energy" />
+			<channel id="power" typeId="power" />
+			<channel id="outlet" typeId="outlet" />
+		</channels>
+
+		<config-description>
+			<parameter name="ain" type="text" required="true">
+				<label>AIN</label>
+				<description>The AHA id (AIN) that identifies one specific device.</description>
+			</parameter>
+		</config-description>
+    </thing-type>
+
+    <thing-type id="FRITZ_Powerline_546E_Solo">
```

Comment:
Ah, ok, you are right - DECT won't work here :-)
So can you hook up 546Es to a 546E?